screens/About.js

```diff
@@ -45,48 +63,40 @@ class Home extends React.Component {
               name="arrow-back"
               color={GLOBAL.COLOR.TOOLBAR_TINT}
               size={30}
-              onPress={() => this.props.navigation.goBack()}
+              onPress={() => navigation.goBack()}
             />
           }
           centerComponent={{
             text: Strings.about,
-            style: { color: GLOBAL.COLOR.TOOLBAR_TINT, fontSize: 18 }
+            style: { color: GLOBAL.COLOR.TOOLBAR_TINT, fontSize: 18 },
           }}
         />
-        <ScrollView
-          style={[
-            styles.container,
-            this.props.nightMode && { backgroundColor: "#000" }
-          ]}
-        >
-          <Text
-            style={[styles.title, this.props.nightMode && styles.nightMode]}
-          >
-            {Strings.sundar_gutka}
-          </Text>
-          <Text
-            style={[styles.small, this.props.nightMode && styles.nightMode]}
-          >
-            {"\n"}{Strings.created_by}:
+        <ScrollView style={[styles.container, nightMode && { backgroundColor: "#000" }]}>
+          <Text style={[styles.title, nightMode && styles.nightMode]}>{Strings.sundar_gutka}</Text>
+          <Text style={[styles.small, nightMode && styles.nightMode]}>
+            {"\n"}
+            {Strings.created_by}:
           </Text>
           <TouchableHighlight
-            underlayColor={"#009bff"}
+            underlayColor="#009bff"
```

Comment:
Veerji please use variables from a common file instead of hard coding colors.

Reply:
fixed